Utils/azure-toolkit-ide-libs/azure-toolkit-ide-vm-lib/src/main/java/com/microsoft/azure/toolkit/ide/vm/VirtualMachineExplorerContributor.java

```diff
@@ -31,16 +29,10 @@
         return new Node<>(service).view(new AzureServiceLabelView<>(service, NAME, ICON))
                 .actions(VirtualMachineActionsContributor.SERVICE_ACTIONS)
                 .addChildren(this::listVirtualMachines, (vm, vmNode) -> new Node<>(vm)
-                        .view(new AzureResourceLabelView<>(vm, AzureResourceLabelView::getResourceIconPath,
-                                Arrays.asList(AzureResourceLabelView::getStatusModifier, VirtualMachineExplorerContributor::getOperatingSystemModifier)))
+                        .view(new AzureResourceLabelView<>(vm))
```

Comment:
how to distinguish vm of different os?

Reply:
Currently we don't support vm with os modifier, which will be add back once our ux update the icon design.